### ISSUE TITLE
Game crashes on startup

### ISSUE THREAD

**Issue by BZSCA** [bluemagic123/tModLoader#4241]
### Version

Latest 1.4.4-stable

### OS

Windows

### Platform

Steam

### This bug affects...

Gameplay as a Player

### Description

on startup, the game does not start and this message appears
![image](https://github.com/tModLoader/tModLoader/assets/13265066/83dd5b0f-eb9b-4b35-9d7c-e1cd5f625523)

### Log File

[environment-client.log](https://github.com/user-attachments/files/15520506/environment-client.log)
[Launch.log](https://github.com/user-attachments/files/15520507/Launch.log)
[Natives.log](https://github.com/user-attachments/files/15520508/Natives.log)
[terrariasteamclient.log](https://github.com/user-attachments/files/15520509/terrariasteamclient.log)
[client.log](https://github.com/user-attachments/files/15520510/client.log)

### Steps to reproduce

1. Attempt to start the game
2. witness issue 

### Expected Behavior

game should start normally

### Actual Behavior

game crashes on startup

### Reproduction frequency

everytime

### Additional Information

there are multiple people experiencing this problem in the discord server on the channel support-forum

**Comment by NotSkittlz** [comment]
I am experiencing the same issue. Here are some of the methods I've tried to use to resolve it:

1) Uninstalling Terraria and TModLoader (and manually deleting remaining files).
2) Restarting Computer.
3) Changing Versions.
4) Verifying the integrity of game files.

The issue persists.

Edit: Just tried switching versions to different betas. v2024.03 was the only version that worked for me. I kept receiving the same issue in versions v2024.04 and v.2024.05.

**Comment by Leeper54** [comment]
Exact same issue. Lot's of people dealing with it.

**Comment by Rijam** [comment]
This is just like the Corsair CUESDK issue. This dll is for the Logitech RGB keyboard support.
Here's a workaround: navigate to that file and delete it. That's it. The game will skip trying to load it if it doesn't exist.

**Comment by BOGUS-TITAN** [comment]
I just had this happen to me as well, exact same error message. Tried all the same stuff as NotSkittlz. Rijam's method worked for me. I went to my file explorer and searched "logitech" in the drive where Terraria and TmodLoader were saved. Two files showed up, one was for terraria and one was for tmodloader. I don't remember what exactly they were called but they had "logitech" in the name. Deleted both of them. Right after that, TmodLoader started up just fine. I still had my character, worlds, items, everything. Not sure how you were able to diagnose this, Mr. Rijam but thank you so much!!! Thanks to you I continue playing one of my favorite games. Much appreciated, brother.
